app/controllers/api/v1/npq_participants_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class NPQParticipantsController < Api::ApiController
+      include ApiTokenAuthenticatable
+      include ApiPagination
+      include ApiFilter
+      rescue_from ActiveRecord::RecordInvalid, with: :invalid_transition
+
+      def index
+        respond_to do |format|
+          format.json do
+            npq_participant_hash = NPQParticipantSerializer.new(paginate(npq_participants)).serializable_hash
+            render json: npq_participant_hash.to_json
+          end
+        end
```

Comment:
Can we remove `respond_to do |format|` until we have to have more than one format here?